packages/context/src/binding-decorator.ts

```diff
@@ -110,3 +113,18 @@ export namespace bind {
     };
   }
 }
+
+/**
+ * Alias decorator `@injectable` to `@injectable` for backward compatibility
+ * @param specs Binding specs
+ */
+export function bind(...specs: BindingSpec[]): ClassDecorator {
+  return injectable(...specs);
+}
+
+/**
+ * Alias namespace `bind` to `injectable` for backward compatibility
+ */
+export namespace bind {
+  export const provider = injectable.provider;
```

Comment:
It makes we wonder if there is away how to export all `injectable` members to avoid the situation where we accidentally miss some?

Woud something like this work?

```ts
export namespace bind {
  export * from injectable;
}
```

Reply:
It does not work. I think it's manageable at this point to hard-code sub items.

Reply:
Fair enough.

What's the rule for adding new `injectable.{something}()` decorators? Are we going to add them to both namespaces (`injectable` and `bind`), or only to the new one (`injectable`)?

It would be great to capture this somewhere in our docs, e.g. as a code comment in this file.